backend/service/topology/cache.go

```diff
@@ -66,3 +69,61 @@ func convertLockIdToAdvisoryLockId(lockID string) uint32 {
 	x := sha256.New().Sum([]byte(lockID))
 	return binary.BigEndian.Uint32(x)
 }
+
+// TODO (mcutalo): make this private once its in use
+func (c *client) SetCache(ctx context.Context, obj *topologyv1.Resource) error {
```

Comment:
Left some todo's but I couldn't keep around private unused func's. In my next PR the will be changed once I implement the CacheableTopology interface.
```
> make backend-lint-fix
tools/golangci-lint.sh run --fix
service/topology/cache.go:114:18: func `(*client).deleteCache` is unused (unused)
service/topology/cache.go:73:18: func `(*client).setCache` is unused (unused)
make: *** [backend-lint-fix] Error 1
```